### DISCUSSION TITLE
Chia - Daily Farming Summary : Incomplete Data - Answered

### DISCUSSION THREAD

**Discussion by k2skaterii** [guydavis/machinaris#236] [Q&A] [answered]
When I look at my Daily Farming Summary it's showing me a Summary from my Harvester w/ 60'ish plots but it's not showing any summary from my Node w/ close to 600 plots.  

I only just converted to Machinaris a few days ago - and the summary was working properly at first...  Maybe it'll automagically fix it's self tomorrow or 2 days from now.   Then again - Maybe not.  

If anyone can provide idea's or guidance about how to troubleshoot why I'm not seeing a summary from my Node - It would be greatly appreciated.  

Thanks!

**Comment by guydavis** [comment, marked as answer]
Hi! Please share a screenshot of your Workers and Summary pages.

**Reply by k2skaterii** [reply]
Please ignore the the plotters & their "connection timeout" / "connection refused" status... That's expected seeing how my plotting VMs are either powered down or the container hasn't been started yet.

chianode is my "node" that is missing from my farming summary page.  

![machinaris-Farming-Summary](https://user-images.githubusercontent.com/87842648/130299809-062bf7c7-58d4-4886-8c72-322ae0e8bd75.PNG)
![machinaris-Workers](https://user-images.githubusercontent.com/87842648/130299811-ac529593-7d58-4de0-9c46-f7a476be3c7c.PNG)

**Comment by k2skaterii** [comment]
Now I'm kinda pissed at myself... I knew I should have just waited.  Less than an hour after I took those screen shots - my problems seems to have resolved itself.

Sorry! 

![image](https://user-images.githubusercontent.com/87842648/130302477-5f0ff8be-5392-4f53-a181-6815cd0e331d.png)

**Reply by k2skaterii** [reply]
So - something is going on here not sure what.  Now I've got 2 summaries from my harvester and one  from my node.

While the problem has changed a bit, one part of my initial post is still valid - "If anyone can provide idea's or guidance about how to troubleshoot  - It would be greatly appreciated!"

![image](https://user-images.githubusercontent.com/87842648/130331414-c4985715-df0b-4c1f-b535-44d4907f08c9.png)

**Reply by guydavis** [reply]
Hi, the screenshots you are showing are the Daily Summaries generated by Chiadog.  Switch to your Alerts page and select all Alerts, delete them, then wait 24 hours and see if the problem recurs when the next set of daily summaries is generated.  Hope this helps!

**Reply by guydavis** [reply]
If this has corrected your concern, please mark this Discussion as Answered.  Thanks!

**Reply by k2skaterii** [reply]
> If this has corrected your concern, please mark this Discussion as Answered. Thanks!

Hey GuyD!  Thank you!  

My issue resolved itself, for the most part by simply waiting.  The other item that came up was that "chia-harv-01" would update its daily stats at 1600 while Chia-node was updating at 2000 - even though both were set to 2000 in the settings / chiadog config.  Restarting the container seems to have solved that issue.

Now everything seems to be working correctly!  

Now, time to move on to expand my machinaris deployment to include containers running on my TrueNAS setup.   With TrueNAS Scale still being in Beta - impressive to see machinaris is one of the 6 applications available from TrueNAS's official charts repo!
